README.rst

```diff
@@ -308,6 +308,9 @@ Keys to take special care are the ones needed to configure Kafka and advertised_
    * - Parameter
      - Default Value
      - Description
+   * - ``aiohttp_client_max_size``
```

Comment:
question: Shouldn't the usage of aiohttp be kept as an internal implementation detail, that we can change without changing names of configuration parameters?

Reply:
`http_request_max_size`.